### PR TITLE
Flip the verifiers and airdrop so claim doesnt need to construct a tr…

### PR DESCRIPTION
**Pull request by brittcyr**
…ansaction

### CODE REVIEW

**Comment by brittcyr** [comment]
Ready to review along with airdrop-sdk change.

File: programs/airdrop/src/instructions/claim.rs [hunk 1]
```diff
@@ -6,10 +6,10 @@ use crate::State;
 use crate::VAULT_SEED;
 
 #[derive(Accounts)]
-#[instruction(amount: u64, verifier_data: Vec<u8>)]
+#[instruction(amount: u64)]
 pub struct Claim<'info> {
-    /// Mutable because might need to pay rent in verifier.
-    #[account(mut)]
+    // TODO: Add one time hack for backwards compatibility of DUAL CSA
```
Comment (DonDuala) [inline programs/airdrop/src/instructions/claim.rs:11]: Do you mean remove? We need backwards compatibility for CSA now right?

File: tests/basic_verifier.ts [hunk 2]
```diff
@@ -1,25 +1,69 @@
...
+        payer: provider.publicKey,
+        state: basicState,
+        vault: basicVault,
+        mint,
+        verifierSignature,
+        tokenProgram: TOKEN_PROGRAM_ID,
+        systemProgram: anchor.web3.SystemProgram.programId,
+        rent: anchor.web3.SYSVAR_RENT_PUBKEY,
+      })
+      .rpc({ skipPreflight: true });
```
Comment (DonDuala) [inline tests/basic_verifier.ts:51]: We want to skip for tests?

File: tests/merkle_verifier.ts [hunk 3]
```diff
@@ -57,31 +99,276 @@ describe('merkle_verifier', () => {
...
+        verifierSignature,
+        tokenProgram: TOKEN_PROGRAM_ID,
+        systemProgram: anchor.web3.SystemProgram.programId,
+        rent: anchor.web3.SYSVAR_RENT_PUBKEY,
+      })
+      .rpc({ skipPreflight: true });
+
+    await mintToAccount(provider, mint, merkleVault, airdropAmount, provider.publicKey);
+
+    const kpOne = anchor.web3.Keypair.generate();
```
Comment (DonDuala) [inline tests/merkle_verifier.ts:175]: Should be unique from L66?

**Review by DonDuala — APPROVED** [review]

File: programs/airdrop/src/instructions/claim.rs (hunk 1 above)
Comment (brittcyr) [reply programs/airdrop/src/instructions/claim.rs:11]: The hack isnt in there yet. will need to bypass for our CSA

File: tests/basic_verifier.ts (hunk 2 above)
Comment (brittcyr) [reply tests/basic_verifier.ts:51]: Doesnt hurt. Makes debugging easier

**Pull request merged**